refactor: fix mypy issues in checkers/h*.py

To see errors run:

`mypy cve_bin_tool/checkers/h*`

The following files are showing errors:

```
cve_bin_tool/checkers/hostapd.py:16: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/haserl.py:15: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/haserl.py:16: error: Need type annotation for "FILENAME_PATTERNS" (hint: "FILENAME_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/haproxy.py:15: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
```

--- a/cve_bin_tool/checkers/haproxy.py
+++ b/cve_bin_tool/checkers/haproxy.py
@@ -9,6 +9,7 @@ https://www.cvedetails.com/product/22372/Haproxy-Haproxy.html?vendor_id=11969
 
 """
 from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 

--- a/cve_bin_tool/checkers/haproxy.py
+++ b/cve_bin_tool/checkers/haproxy.py
@@ -8,11 +8,12 @@ CVE checker for haproxy
 https://www.cvedetails.com/product/22372/Haproxy-Haproxy.html?vendor_id=11969
 
 """
+from typing import List
 from cve_bin_tool.checkers import Checker
 
 
 class HaproxyChecker(Checker):
-    CONTAINS_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
     FILENAME_PATTERNS = [r"haproxy"]
     VERSION_PATTERNS = [
         r"HA-Proxy version ([0-9]+\.[0-9]+\.[0-9]+)",

--- a/cve_bin_tool/checkers/haserl.py
+++ b/cve_bin_tool/checkers/haserl.py
@@ -8,11 +8,12 @@ CVE checker for haserl
 https://www.cvedetails.com/product/82440/Haserl-Project-Haserl.html?vendor_id=23119
 
 """
+from typing import List
 from cve_bin_tool.checkers import Checker
 
 
 class HaserlChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = ["haserl version ([0-9]+\\.[0-9]+\\.[0-9]+)"]
     VENDOR_PRODUCT = [("haserl_project", "haserl")]

--- a/cve_bin_tool/checkers/haserl.py
+++ b/cve_bin_tool/checkers/haserl.py
@@ -9,6 +9,7 @@ https://www.cvedetails.com/product/82440/Haserl-Project-Haserl.html?vendor_id=23
 
 """
 from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 

--- a/cve_bin_tool/checkers/hostapd.py
+++ b/cve_bin_tool/checkers/hostapd.py
@@ -8,12 +8,13 @@ CVE checker for hostapd
 https://www.cvedetails.com/product/22495/W1.fi-Hostapd.html?vendor_id=12005
 
 """
+from typing import List
 from cve_bin_tool.checkers import Checker
 
 
 class HostapdChecker(Checker):
     # FIXME: fix contains pattern
-    CONTAINS_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
     FILENAME_PATTERNS = [r"hostapd"]
     VERSION_PATTERNS = [r"hostapd v([0-9]+\.[0-9]+)"]
     VENDOR_PRODUCT = [("w1.fi", "hostapd")]

--- a/cve_bin_tool/checkers/hostapd.py
+++ b/cve_bin_tool/checkers/hostapd.py
@@ -9,6 +9,7 @@ https://www.cvedetails.com/product/22495/W1.fi-Hostapd.html?vendor_id=12005
 
 """
 from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 

